next.config.js

```diff
@@ -8,6 +8,14 @@ const nextConfig = {
   typescript: {
     ignoreBuildErrors: true,
   },
+  async rewrites() {
+    return [
+      {
+        source: '/api/:path*',
+        destination: 'http://localhost:8000/*',
+      },
+    ]
+  },
```

Comment:
i didn't know we could this. is there a way to make this dynamic based on current environment?